01 - Community/People/avr.md

```diff
@@ -10,7 +10,7 @@ publish: true
 
 - GitHub: [avr](https://github.com/avr/) ^github
 <!-- - Discord: `@` ^discord-->
-<!-- - Website: <> ^website-->
+- Website: <https://github.com/avr> ^website
```

Comment:
That‘s the same as the GitHub address.

Reply:
Good spot.

**This case**

The `update-releases.py` script gathers author Urls from the `authorUrl` values in all the manifest.json files in the author's plugins.

It then picks one of them, and uses that.

I checked, and `avr` has one plugin currently. Its manifest has this line:

https://github.com/avr/obsidian-reading-time/blob/ed951ccc7d26a99f0ed746b28d4acc208c3d37dc/manifest.json#L9

> `"authorUrl": "https://github.com/avr"`

Which they must have added since the page was first created.

So I think that this change reflects the author's intent.

**Other cases**

As an aside, not all plugin authors use that field consistently across all their repos... there are a few prolific plugin authors who put the Plugin's Url in the authorUrl field. This means that as they add new plugins, the generated url changes... and it's the (undocumented) convention of this project to revert those scripted edits...

The attached file shows all the author URLs that point in to GitHub.com:

[github-account-as-author-url.txt](https://github.com/obsidian-community/obsidian-hub/files/7846072/github-account-as-author-url.txt)

Reply:
I've added https://github.com/obsidian-community/obsidian-hub/issues/217 as a way of preventing the need for this manual step:

> and it's the (undocumented) convention of this project to revert those scripted edits...